Ross_Hargrove/CoinTosses.py

```diff
@@ -0,0 +1,19 @@
+import random
+
```

Comment:
I like the comments here.  That can be really helpful if you step away from a codebase for a while and then go back to it.